$NetBSD: patch-aa,v 1.2 2011/11/07 23:00:48 othyro Exp $

Making install in po cannot find install.sh. Fixes that issue.

--- po/Makefile.in.in.orig	2008-02-06 09:27:54.000000000 +0000
+++ po/Makefile.in.in
@@ -28,6 +28,7 @@ exec_prefix = @exec_prefix@
 datadir = @datadir@
 libdir = @libdir@
 mkdir_p=@mkdir_p@
+top_builddir = @top_builddir@
 localedir = $(libdir)/locale
 gnulocaledir = $(datadir)/locale
 gettextsrcdir = $(datadir)/glib-2.0/gettext/po
